src/index.ts

```diff
@@ -43,19 +43,22 @@ const compileTemplate = async (name: string, schema: types.OpenRPC, language: st
   });
 };
 
+const moveFiles = async (dirName: string, file1: string, file2: string) => {
+  try {
```

Comment:
might have been an oversight to `try catch` & do nothing. I think it would be a reasonable improvement to throw with a good error message instead of doing nothing.

Reply:
@BelfordZ Well it throws an error every time because there are because the "rs" template doesn't have the `.gitignore` or "package.json".

I could do something like this 

```javascript
if(dirname === "js") {
  const moveFiles = async (dirName: string, file1: string, file2: string) => {
    try {
      await move(path.join(dirName, file1), path.join(dirName, file2));
    } catch (error) {
      // do nothing
      throw new Error(error);
    }
  }
}
```

Reply:
Ah right. I remember now, thank you.

if I remember correctly, I had to put _package.json so that npm wouldn't think it should install those. I think we can solve this better by adding the templates directory to the root `.npmignore`, and then change the _package.json file name back to just package.json.

For the dotfiles, IMO it would be best to use just write a copy function that doesn't ignore files with prefixed with `.`. Or perhaps just use https://github.com/AvianFlu/ncp instead of fs-extra

Reply:
That being said, this **does** remove a bug, and therefor, IMO we should get this in while we work on a cleaner solution.